src/components/drawing/index.js

```diff
@@ -512,32 +513,36 @@ drawing.bBox = function(node) {
         return Lib.extendFlat({}, savedBBoxes[saveNum.value]);
     }
 
-    var test3 = d3.select('#js-plotly-tester'),
```

Comment:
Not sure if I understand correctly. I did cache the tester div in line 517 and 518 already. Is that what you are referring to?